src/worldcereal/openeo/preprocessing.py

```diff
@@ -236,16 +250,15 @@ def worldcereal_preprocessed_inputs_gfmap(
         spatial_extent=spatial_extent,
         temporal_extent=temporal_extent,
         bands=[
-            "S1-SIGMA0-VV",
             "S1-SIGMA0-VH",
+            "S1-SIGMA0-VV",
         ],
         fetch_type=FetchType.TILE,
         target_resolution=10.0,  # Compute the backscatter at 20m resolution, then upsample nearest neighbor when merging cubes
         orbit_direction="ASCENDING",
```

Comment:
This is artefact from before. We shouldn't be hardcoding this but choose an orbit dynamically based on availability of observations.